src/lib/wordpress-provider/playground-cli/playground-cli-provider.ts

```diff
@@ -115,26 +112,42 @@ export class PlaygroundCliProvider implements WordPressProvider {
 		return '/wordpress/wp-load.php';
 	}
 
+	private escapePhpString( str: string ): string {
+		return str.replace( /\\/g, '\\\\' ).replace( /'/g, "\\'" );
+	}
+
 	async setupWordPressSite( server: SiteServer, wpVersion = 'latest' ): Promise< boolean > {
-		const { path, port, adminPassword, name, phpVersion } = server.details;
-		const { blueprint } = server.meta;
-		const siteLanguage = await getPreferredSiteLanguage( wpVersion );
-		const serverOptions = {
-			path,
-			port,
-			adminPassword: adminPassword || 'password',
-			siteTitle: name,
-			phpVersion: phpVersion || this.DEFAULT_PHP_VERSION,
-			wpVersion,
-			isWpAutoUpdating: false,
-			isSetupMode: true,
-			blueprint: blueprint?.blueprint,
-			siteLanguage,
-		};
-		let serverProcess;
+		const { path, name, adminPassword } = server.details;
 
 		try {
 			const isOnline = net.isOnline();
+			const siteLanguage = await getPreferredSiteLanguage( wpVersion );
+
+			const setupSteps = [];
```

Comment:
Let's define the steps type.

```suggestion
			const setupSteps: StepDefinition[] = [];
```